source/features/bugs-tab.tsx

```diff
@@ -24,14 +31,45 @@ async function highlightBugsTabOnIssuePage(): Promise<void | false> {
 	issuesTab.removeAttribute('aria-current');
 }
 
+const cacheKey = (): string => 'bugs-label:' + getRepo()!.nameWithOwner;
+
 const countBugs = cache.function(async (): Promise<number> => {
-	const {search} = await api.v4(`
-		search(type: ISSUE, query: "label:bug is:open is:issue repo:${getRepo()!.nameWithOwner}") {
-			issueCount
+	if (!await cache.get(cacheKey())) {
```

Comment:
You're getting this value several times in this function. Extract it to its own variable. Also you can use <string> to specify its type instead of String()